### PR TITLE
Homework Lesson 8

### PR DESCRIPTION
**Pull request by Andrey-Kalashnikov**

### CODE REVIEW

File: 8-CSS-Position-and-Transform/Part-1/index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/style.css" />
+  <link rel="stylesheet" href="./css/all.min.css" />
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet" />
+  <title>Hostel</title>
+</head>
+
+<body>
+  <img class="background" src="./img/bg.png" alt="" />
```
Comment (idovbush) [inline 8-CSS-Position-and-Transform/Part-1/index.html]: this is our header and all its element must be wrapped with one section or div or <header>

File: 8-CSS-Position-and-Transform/Part-1/index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  <img class="mask" src="./img/mask.png" alt="" />
+  <img class="logo" src="./img/logo.png" alt="UPG logo" />
```
Comment (idovbush) [inline 8-CSS-Position-and-Transform/Part-1/index.html]: this is logo and it includes two element (img and title) so they also should be wrapped together

File: 8-CSS-Position-and-Transform/Part-1/index.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  <div class="UDG">URBAN DEVELOPMENT GROUP</div>
+  <nav>
+    <div class="search">
```
Comment (idovbush) [inline 8-CSS-Position-and-Transform/Part-1/index.html]: I think the order of this element shoudl be reversed (ul, lang, search)

File: 8-CSS-Position-and-Transform/Part-1/css/style.css [hunk 4]
```diff
@@ -0,0 +1,218 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.background {
+  z-index: -2;
+  position: fixed;
```
Comment (idovbush) [inline 8-CSS-Position-and-Transform/Part-1/css/style.css:8]: it's shoudl be fixed cause in a while we will continue work on that whole mockup and when you will scroll it you will not see new elements cause this element is fixed

File: 8-CSS-Position-and-Transform/Part-2/css/style.css [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+    font-size: 80px;
+}
+
+p {
+    margin-top: 30px;
+    font-size: 17px;
+}
+
+section { 
+    position: absolute;
```
Comment (idovbush) [inline 8-CSS-Position-and-Transform/Part-2/css/style.css]: redundant usage of position absolute, we don't have a reason apply that to it

File: 8-CSS-Position-and-Transform/Part-1/css/style.css (hunk 4 above)
Comment (Andrey-Kalashnikov) [reply 8-CSS-Position-and-Transform/Part-1/css/style.css:8]: am unlock him in the future

**Pull request merged**